daemon/algod/api/algod.oas3.yml

```diff
@@ -1027,9 +1027,14 @@
             "type": "string"
           },
           "name": {
-            "description": "\\[an\\] Name of this asset, as supplied by the creator.",
+            "description": "\\[an\\] Name of this asset, as supplied by the creator. Field presents only if the asset name is composed of printable utf-8 characters.",
```

Comment:
I suggest saying "Available only if ..." instead of "Field presents only if ..."

Reply:
There are several places in the current descriptions with phrasing like `This is only included when`, and `This endpoint is only enabled when`